docs/specs/localization.yml

```diff
@@ -52,6 +52,74 @@ repos:
 outputs:
   docs/dep.json:
 ---
+# restore source repo when build from loc repo directly 1
```

Comment:
Add description to avoid 1,2,3?

Reply:
description is in second line, the first line is helping me to locate the failed test case quickly

Reply:
ok